core/dispatcher/src/main/scala/whisk/core/dispatcher/ActivationFeed.scala

```diff
@@ -83,7 +84,12 @@ protected class ActivationFeed(
                             handler(topic, bytes)
                     }
                 }
-                consumer.commit()
+                Try {
+                    consumer.commit()
+                } recover {
+                    case e: CommitFailedException => logging.warn(this, s"Error on committing to Kafka ${e.getMessage}")
+                    case e: Throwable             => logging.error(this, s"Unexpected error on committing to Kafka: ${e.getMessage}")
+                }
                 fill()
```

Comment:
note that a fill without a commit will cause messages to be replayed - this will violate at most once semantics.